error-prone-contrib/src/test/resources/tech/picnic/errorprone/refasterrules/NullRulesTestInput.java

```diff
@@ -30,11 +33,15 @@ String testRequireNonNullElseGet() {
     return Optional.ofNullable("foo").orElseGet(() -> "bar");
   }
 
-  long testIsNullFunction() {
-    return Stream.of("foo").filter(s -> s == null).count();
+  ImmutableSet<Long> testIsNullFunction() {
+    return ImmutableSet.of(
+        Stream.of("foo").filter(s -> s == null).count(),
+        Stream.of("bar").filter(not(Objects::nonNull)).count());
   }
 
-  long testNonNullFunction() {
-    return Stream.of("foo").filter(s -> s != null).count();
+  ImmutableSet<Long> testNonNullFunction() {
+    return ImmutableSet.of(
+        Stream.of("foo").filter(s -> s != null).count(),
+        Stream.of("bar").filter(not(Objects::isNull)).count());
   }
```

Comment:
I see that these tests were a bit more complicated than how we usually write them; will use this opportunity to simplify.